pkg/catalogv2/system/system.go

```diff
@@ -123,26 +123,22 @@ func (m *Manager) installCharts(charts map[desiredKey]map[string]interface{}) {
 }
 
 func (m *Manager) Uninstall(namespace, name string) error {
-	helmcfg := &action.Configuration{}
```

Comment:
Why remove all the list/check code ? The install method below still checks that inside `isPendingUninstall`, which I'm assuming is there so it doesn't error out. Must be some reason it existed.